package.json

```diff
@@ -26,7 +26,7 @@
     "clsx": "1.2.1",
     "docusaurus-plugin-hotjar": "0.0.2",
     "lodash": "4.17.21",
-    "lottie-web": "5.10.2",
+    "lottie-react": "^2.4.0",
```

Comment:
Can we fix the version as it was before? You can always use `--save-exact` or `-E` flag when installing dependencies ([ref](https://docs.npmjs.com/cli/v8/commands/npm-install#description))